### DISCUSSION TITLE
Select the chatgpt template to use and be able to insert attachments

### DISCUSSION THREAD

**Discussion by newtec-foggia** [Ercogx/filament-openai-assistant#5] [Ideas]
### What happened?

it would be interesting to be able to select the chatgpt template to use and be able to insert attachments

### How to reproduce the bug

install the latest version

### Package Version

1.0.4

### PHP Version

8.2.0

### Laravel Version

10.48.17

### Which operating systems does with happen with?

Windows

### Notes

very interesting plugin

**Comment by Ercogx** [comment]
I don't quite understand which templates we are talking about. 

About attachments, there are plans to add functionality with this, but unfortunately there is no time to do it yet

**Reply by newtec-foggia** [reply]
![modello_chatgpt](https://github.com/user-attachments/assets/23adf6b0-4f19-4c30-a758-433fd6f6efe0)

**Reply by Ercogx** [reply]
Are you talking about configuring the assistant through the UI? 

I don't know how often this is a case. I usually set up assistants 1 time per project. If you want, you can make a pull request with this functionality

**Comment by Ercogx** [comment]
I opened a discussion and moved it here, probably more relevant
